doc/openOCD.md

```diff
@@ -27,7 +27,7 @@ sudo cp contrib/60-openocd.rules /etc/udev/rules.d/
 sudo udevadm control --reload-rules
 ```
 
- - You can now plug your STLinkV2 in a USB port and run OpenOCD to see if it's working correctly:
+ - You can now plug your STLinkV2 into a USB port and run OpenOCD to see if it's working correctly:
```

Comment:
Probably should be "an USB" as well?

Reply:
I don't think so, because I believe using "an" or "a" [depends on the sound of the following word](https://www.merriam-webster.com/words-at-play/is-it-a-or-an). In this case since "USB" is an acronym pronounced "yoo-es-bee" I believe it would be "a". "an USB" also doesn't flow well and would draw attention back to the phrasing instead of the informational content, which we are trying to avoid.